core/backend/src/IModelHost.ts

```diff
@@ -265,9 +264,6 @@ export class IModelHost {
   /** The AuthorizationClient used to obtain [AccessToken]($bentley)s. */
   public static authorizationClient?: AuthorizationClient;
 
-  /** @alpha */
-  public static readonly telemetry = new TelemetryManager();
```

Comment:
did you also want to remove IModelApp.telemetry? Already marked as internal and doesnt seem to be used

Reply:
IModelApp unfortunately is the one that is being used by AppUI so if it is removed they would need to update their minimum core supported.